fxgl/src/main/kotlin/com/almasb/fxgl/app/Settings.kt

```diff
@@ -235,6 +236,14 @@ class GameSettings(
         var achievements: List<Achievement> = arrayListOf()
 ) {
 
+    public fun setHeightFromRatio(ratio: Double) {
```

Comment:
1. `public` is redundant, it's `public` by default. 
2. looks good, but unit tests are needed. You can access these new methods from [this line](https://github.com/AlmasB/FXGL/blob/dev/fxgl/src/test/kotlin/com/almasb/fxgl/app/GameSettingsTest.kt#L60). So first set width to a value, then call setHeightFromRatio and check the result. Then do the same with height. Finally, restore width and height to 500 so the other tests pass

Reply:
Changed the base to dev.
Forgot to remove the public keyword, it was a mistake.
I'll add the unit tests and commit.

Reply:
Actually, scratch that last comment. It would be better to create one new test function:

```
fun `Test set ...`() {
 val settings = GameSettings()
// then do the test
}
```

Reply:
And i don't want to pollute beforeAll for testing this. I'll write as a separate test and initiate settings once again there if that's okay.

Reply:
Ignore my last reply too, explained the same without noticing your last reply.